### DISCUSSION TITLE
Improve "Reference" by adding default values consistently

### DISCUSSION THREAD

**Discussion by ypislon** [quarto-dev/quarto-cli#5098] [Feature Requests]
Dear quarto team,
as a new user of Quarto I would appreciate a more consistent documentation in the "reference" sections on the website by noting what the default values are for all options. In most cases, the default values are explicitly mentioned, but in some, they are not. This would help when diving deeper into the configuration and setup of Quarto projects.

**Comment by ghost** [comment]
Thanks for posting this.  Can you please give us some examples of which ones are missing that you would have liked to have seen?

**Reply by ypislon** [reply]
I am using Quarto with the goal to render a website, so I can name some examples from the current ["Website" reference](https://quarto.org/docs/reference/projects/websites.html) section:

Property `website: navbar: search`, Description: "Include a search box in the navbar.". Desired: "Include a search box in the navbar (default: `false`)."
Property `website: navbar: pinned`, Description: "Always show the navbar (keeping it pinned)." Desired: "Always show the navbar (keeping it pinned) (default: `false`)."
Property `project: output-dir`, Description: "Output directory", Desired: "Output directory (default: `_site`)" 

Same for other sections, to name a few examples in the OJS section:
[OJS, code output](https://quarto.org/docs/reference/cells/cells-ojs.html#code-output): 
Property `code-overflow`, Description: "Choose how to handle code overflow, when code lines are too wide for their container. One of: scroll, wrap". Desired: "Choose how to handle code overflow, when code lines are too wide for their container. One of: scroll (default), wrap"
[OJS, cell output](https://quarto.org/docs/reference/cells/cells-ojs.html#cell-output):
In this section of configuration of cell output, no default values are described (for `true`/`false` toggles, but also for properties like `panel` with multiple options).

Documenting the default values for all options would make the developer experience easier when setting up a project with specific needs. Furthermore I understand the "Reference" to be a place for documentation with no space for examples, so having default values would useful to know "what you're getting" without a visual example like in the guides. This seems to be best practice in other frameworks as well, e.g. in Bootstrap.

**Reply by ghost** [reply]
Thank you for walking us through some of these concrete examples and some of the challenges you had with understanding the default behaviour.  I'm going to to move this to #5109.  I've also included @cwickham to help review.

**Comment by ghost** [comment]
Moved to #5109
